docs/spark-procedures.md

```diff
@@ -421,12 +421,17 @@ Existing data files are added to the Iceberg table's metadata and can be read us
 
 To leave the original table intact while testing, use [`snapshot`](#snapshot) to create new temporary table that shares source data files and schema.
 
+By default, the original table is retained with the name `table_BACKUP_`. You can also explicitly pass  `drop_backup => true` 
+to drop the backup after migration finishes.
+
 #### Usage
 
 | Argument Name | Required? | Type | Description |
 |---------------|-----------|------|-------------|
 | `table`       | ✔️  | string | Name of the table to migrate |
-| `properties`  | ️   | map<string, string> | Properties for the new Iceberg table |
+| `properties`  | ️   | map<string, string> | Properties for the new Iceberg table | 
```

Comment:
Sorry for the multiple commits. I am unable to figure out what changed in this line. Seems exactly the same to my eyes. Something hidden character must have been added somehow.

Reply:
Do you have one space after the last | ?  You can maybe just copy and paste from the previous version?

Reply:
Yeah, I think there was an extra space after | . I removed it in the latest commit.